wasm/jit/jit_arm64.go

```diff
@@ -1,15 +1,39 @@
 //go:build arm64
 // +build arm64
 
+// This file implements the compiler for arm64 target.
+// Please refer to https://developer.arm.com/documentation/102374/latest/
+// if unfamiliar with arm64 instructions and semantics.
+//
+// Note: we use arm64 pkg as the assembler which has different notation
```

Comment:
by arm64 package you mean "github.com/twitchyliquid64/golang-asm/obj/arm64"? if so, break out this import and add this comment to it?